src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```diff
@@ -0,0 +1,58 @@
+# approved scripts
+metadataGatheringDisabled=Metadata gathering disabled. \
+  <p>You can re-enable it permanently by removing the system property: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.metadataGathering</code> </p>\
+  <p>or temporarily (until restart) by running this in the Script Console: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.METADATA_GATHERING = true</code></p>
+
+## table headers
+approvedExpandCode=Code
+approvedLanguage=Language
+usageCount=# of uses
+usageCount_tooltip=How often this script was executed. Only executions since Script Security plugin was updated to version 1.75.
+lastTimeUsed=Date of last use
+wasPreapproved_header_tooltip=The presence of a green tick means the script was approved during the configuration. \
+  Otherwise it was created by a user without the permission to run scripts and was then approved using this page.
+lastApprovalTime=Date of the last approval
+lastApprovalTime_tooltip=A single script can be approved multiple times. It occurs every time the related configuration is saved.
+lastKnownApproverLogin=Last approver
+
+## table content
+noRequesterSinceMetadata=N/A
+noRequesterSinceMetadata_tooltip=The requester was not recorded. \
+  It could be because the request was created before the metadata was introduced or the code calling this script did not provide user information.
+noContextItemSinceMetadata=N/A
+noContextItemSinceMetadata_tooltip=The context item was not provided by the code asking for approval.
+
+emptyMetadata=There is no metadata for this approval.
+emptyMetadata_tooltip=If the script is used after the metadata introduction, some metadata will be displayed here. \
+  It means that an unused approval will not have metadata and could be reasonably removed after a certain period of time.
+noScriptCodeSinceMetadata=N/A
+noScriptCodeSinceMetadata_tooltip=The approval was given while the metadata was not gathered.
+approvedContextUser=Requester
+approvedContextItem=Context
+wasPreapproved_tooltip=The script content was approved directly during configuration, meaning it comes from a user with the permission to run scripts.
+wasPreapproved_alt=Preapproved
+noUsageCountSinceMetadata=N/A
+noUsageCountSinceMetadata_tooltip=It was not used since the introduction of metadata.
+notUsedSinceMetadata=N/A
+notUsedSinceMetadata_tooltip=It was not used since the introduction of metadata.
+notApprovedSinceMetadata=N/A
+notApprovedSinceMetadata_tooltip=It was not approved since the introduction of metadata.
+noKnownApproverSinceMetadata=N/A
+noKnownApproverSinceMetadata_tooltip=It was not approved since the introduction of metadata.
```

Comment:
```suggestion
noUsageCountSinceMetadata_tooltip=It has not been used since the introduction of metadata.
notUsedSinceMetadata=N/A
notUsedSinceMetadata_tooltip=It has not been used since the introduction of metadata.
notApprovedSinceMetadata=N/A
notApprovedSinceMetadata_tooltip=It has not been approved since the introduction of metadata.
noKnownApproverSinceMetadata=N/A
noKnownApproverSinceMetadata_tooltip=It has not been approved since the introduction of metadata.
```